packages/front-end/styles/theme.scss

```diff
@@ -41,6 +41,10 @@
   --alert-warn-border-color: #ffeeba;
   --alert-warn-color: #856404;
 
+  --alert-caution-background: #ffe2cd;
+  --alert-caution-border-color: #eac5af;
+  --alert-caution-color: #d06703;
```

Comment:
I see these are defined but they don't appear to be used. The highlight colour in settings is still using the hardcoded values.